robot/basestation/app.py

```diff
@@ -205,54 +204,61 @@ def rover_drive():
 
     return jsonify(success=True, cmd=cmd, feedback=feedback, error=error)
 
-# capture image
-@app.route("/capture_image", methods=["POST", "GET"])
+@app.route("/capture_image/", methods=["POST", "GET"])
 def capture_image():
```

Comment:
To help with making things more modular, we should move the main logic of this function into another python file. Like how the stream recording functions are. Maybe stream_capture.py would be an appropriate place for it?

Reply:
done